scripts/presets/travis/sharded_clusters/cluster.json

```diff
@@ -0,0 +1,89 @@
+{
+    "configsvrs": [ {
+		"members" : [
```

Comment:
These JSON files appear to be using a mix of four-space and tab indents. I'd suggest standardizing on four-space indents, as I did for our non-Travis configs in https://github.com/mongodb/mongo-php-driver/commit/8b19fcbc152af9fa9c2e0e67b34e4b348f2ff2d9.

Reply:
It was only this one... → fixed.